py/desitarget/subpriority.py

```diff
@@ -62,13 +64,13 @@ def get_fiberassign_subpriorities(fiberassignfiles):
     """
     log = get_logger()
 
-    #- allow duplicate inputs, but don't process multiple tiles
+    # - allow duplicate inputs, but don't process multiple tiles
```

Comment:
FWIW, I suspect that some style checker converted all my `#-` comments into `# -`.  You don't need to use my style in desitarget and I try to follow your `# INITIALS` style in desitarget when I'm thinking of it, but sometimes habit slips through.  The basic idea of `#-` is that it distinguishes genuine comments from `# ` which I use for commenting out code.  In practice, I'm the only person who follows that in desi code so `#-` and now `# -` defacto means `# SB`.  :)

Reply:
That's fine and understood. I'm just trying to (pedantically) pass the PEP 8 style-checker as much as possible. One PEP 8 check is that there's a space after every comment.